test/dtlstest.c

```diff
@@ -216,7 +216,7 @@ static int test_dtls_drop_records(int idx)
                                        &sctx, &cctx, cert, privkey)))
         return 0;
 
-#ifdef OPENSSL_NO_DTLS1_2
+#if defined(OPENSSL_NO_DTLS1_2) && defined(OPENSSL_NO_DTLS1_3)
```

Comment:
This isn't correct because the `create_ssl_ctx_pair` above explicitly restricts this to DTLSv1.2 only

Reply:
Good catch. I was running on autopilot..